src/css/layout/_l-hero.scss

```diff
@@ -86,5 +81,9 @@
 		@include mappy-bp(lap-small) {
 			margin: 0;
 		}
+
+		@include mappy-bp(lap-large) {
+			margin-bottom: rem(50);
```

Comment:
This is meant to better position the numeronym header content closer to what the current deployment looks like.